sdk/core/amqp-common/src/ConnectionContextBase.ts

```diff
@@ -162,7 +156,8 @@ export module ConnectionContextBase {
         platform: `(${os.arch()}-${os.type()}-${os.release()})`,
         framework: `Node/${process.version}`
       },
-      operationTimeoutInSeconds: parameters.operationTimeoutInSeconds
+      operationTimeoutInSeconds: parameters.operationTimeoutInSeconds,
+      idle_time_out: Constants.defaultConnectionIdleTimeoutInMs
```

Comment:
idle_time_out is probably not our field, is it? If it is our field - where'd that snake_case come from? :)

Reply:
This comes straight from the RheaConnectionOptions in `rhea`. :)

Reply:
The service (both Event Hubs & Service Bus) have this feature where they disconnect the connection if it has been idle for more than 60 seconds. I want to make sure we understand the difference between that scenario and this.